ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/dot1x.md

```diff
@@ -43,3 +43,18 @@ interface Management1
 | System Auth Control | Protocol LLDP Bypass | Dynamic Authorization |
 | ------------------- | -------------------- | ----------------------|
 | True | True | True |
+
+
+
```

Comment:
Something is not correct in the template, since we render these blank lines.

Reply:
Fixed